src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -0,0 +1,410 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Generic Sensor API is available for Origin Trials in Chome 62.
+
+{# wf_updated_on: 2017-09-20 #}
+{# wf_published_on: 2017-09-20 #}
+{# wf_tags: sensors,origintrials,chrome62,news #}
+{# wf_blink_components: Blink>Sensor #}
+{# wf_featured_image: /web/updates/images/generic/sd-card.png #}
+{# wf_featured_snippet: Generic Sensor API is available for Origin Trials in Chome 62. #}
+
+# Sensors For The Web! {: .page-title }
+
+{% include "web/_shared/contributors/alexshalamov.html" %}
+{% include "web/_shared/contributors/pozdnyakov.html" %}
+
+Today, sensor data is used in many native applications to enable use cases such
+as immersive gaming, fitness tracking, and augmented or virtual reality. Wouldn't
+it be cool to bridge the gap between native and web applications? The [Generic Sensor
+API](https://www.w3.org/TR/generic-sensor/), FTW!
+
+## What is Generic Sensor API? {: #what-is-generic-sensor-api }
+
+The [Generic Sensor API](https://www.w3.org/TR/generic-sensor/) is a set of
+interfaces which expose sensor devices to the web platform. The API consists
+of the base [Sensor](https://w3c.github.io/sensors/#the-sensor-interface)
+ interface and a set of concrete sensor classes built on top. Having a base
+interface simplifies the implementation and specification process for the
+concrete sensor classes. For instance, take a look at the
+[Gyroscope](https://w3c.github.io/gyroscope/#gyroscope-interface)
+ class, it is super tiny! The core functionality is specified by the base
+interface, and Gyroscope merely extends it with three attributes representing
+angular velocity.
+
+Typically a concrete sensor class represents an actual sensor on the platform
+e.g., accelerometer or gyroscope. However, in some cases, implementation of a
+sensor class [fuses](https://w3c.github.io/sensors/#sensor-fusion) data from
+several platform sensors and exposes the result in a convenient way to the user.
+For example, the
+[AbsoluteOrientation](https://www.w3.org/TR/orientation-sensor/#absoluteorientationsensor)
+ sensor provides ready-to-use 4x4 rotation matrix based on the data obtained
+from accelerometer, gyroscope and magnetometer.
+
+You might think that the web platform already provides sensor data and you are
+absolutely right! For instance, [DeviceMotion](https://www.w3.org/TR/2016/CR-orientation-event-20160818/#devicemotion)
```

Comment:
Please point to our reference docs on these at [https://developers.google.com/web/fundamentals/native-hardware/device-orientation/](https://developers.google.com/web/fundamentals/native-hardware/device-orientation/)